### ISSUE TITLE
Formatting `<style>` tag flattens css

### ISSUE THREAD

**Issue by zkat** [surface-ui/surface_formatter#66]
The contents of the `style` tag should probably be run through a reasonable css formatter. Right now, they're just flattened to the left:

![20240324_18h31m45s_grim](https://github.com/surface-ui/surface_formatter/assets/17535/7e666b07-5d22-4388-b3b5-9b126e81d59e)

**Comment by paulstatezny** [comment]
PRs are welcome :)

**Comment by msaraiva** [comment]
Hey @zkat!

Thanks for reporting this issue. I believe the surface formatter shouldn't touch anything inside `<style>` as we don't want to keep a full-featured css parser and formatter inside Surface. We should treat it just like a macro component and ignore any of its content. Then you could probably set up your own formatter/linter, something like Prettier to do that job for you.

@paulstatezny I believe the fix would be to add a clause for `style` [here](https://github.com/surface-ui/surface/blob/cf7ac18513d0a054da239bb2b81f0cf6df0404fb/lib/surface/formatter.ex#L427), right?

**Comment by paulstatezny** [comment]
@msaraiva Yes, adding a clause there would prevent the formatter from modifying the contents of `<style>` tags.
